models/adwords/model.sql

```diff
@@ -0,0 +1,26 @@
+create or replace view {schema}.adwords_summary as (
+  with ad_data as
+    (
+    select 
+      lower(addestinationurl) as cleanurl,
+      * 
+    from 
+      adwords.adwords12218869_v2 -- How do I make this work regardless of the extension?
```

Comment:
Chris worked on a solution to this problem here: https://github.com/analyst-collective/models/pull/8

Ideally, it would look like `adwords.adwords{adwords_id}` (or something) and the `{adwords_id}` would be interpolated by the runner at "compile time"